jaseci_kit/jaseci_kit/modules/text_seg/text_seg.py

```diff
@@ -67,7 +67,7 @@ def get_segments(text: str, threshold: float = 0.7):
 
 
 @jaseci_action(act_group=["text_seg"], allow_remote=True)
-def load_model(model_name: str):  # modelname could be ("wiki", "legal")
+def seg_load_model(model_name: str):  # modelname could be ("wiki", "legal")
```

Comment:
Why does this need to be changed?

Reply:
As discussed, this is for parallel execution issue, we may have to find a better solution for this